lms/templates/ccx/schedule.html

```diff
@@ -124,5 +124,18 @@
         'dateFormat': 'yy-mm-dd',
         'autoclose': true
     });
+    $('.datepair .date').change(function() {
```

Comment:
Why aren't we just using the datepickers methods for grabbing the date object? https://api.jqueryui.com/datepicker/